git/client.go

```diff
@@ -489,6 +489,31 @@ func (c *Client) AddRemote(ctx context.Context, name, urlStr string, trackingBra
 	return remote, nil
 }
 
+func (c *Client) InGitDirectory(ctx context.Context) bool {
+	showCmd, err := c.Command(ctx, "rev-parse", "--is-inside-work-tree")
+	if err != nil {
+		return false
+	}
+	out, err := showCmd.Output()
+	if err != nil {
+		return false
+	}
+
+	split := strings.Split(string(out), "\n")
+	if len(split) > 0 {
+		return split[0] == "true"
+	}
+	return false
+}
+
+func (c *Client) UnsetRemoteResolution(ctx context.Context, name string) error {
+	unsetCmd, err := c.Command(ctx, "config", "--unset", fmt.Sprintf("remote.%s.gh-resolved", name))
+	if err != nil {
+		return err
+	}
+	return unsetCmd.Run()
```

Comment:
```suggestion
	args := []string{"config", "--unset", fmt.Sprintf("remote.%s.gh-resolved", name)}
	cmd, err := c.Command(ctx, args...)
	if err != nil {
		return err
	}
	_, err = cmd.Output()
	if err != nil {
		return err
	}
	return nil
```

Since iostreams are attached to the command by default with the git client, using `Run()` here will result in the user seeing the output from this command. What do you think about following the `Output()` pattern from `SetRemoteResolution`?